interfaces/builtin/tee.go

```diff
@@ -0,0 +1,62 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
```

Comment:
2021? I guess that PR has been open since 2020 but only now is ripe for landing? :smile: